### ISSUE TITLE
Slow odometry

### ISSUE THREAD

**Issue by cdondrup** [strands-project/scitos_drivers#41]
Does anyone know why the odometry is only published at about 10hz. This seems very slow to me. It would be nice if it could be at least as fast as the kinect (~30hz) otherwise the odometry would be the bottleneck and not the calculation heavy vision magic which would be very odd.
Is that some kind of mira related problem or is it maybe just a problem on Linda?

**Comment by cburbridge** [comment]
I set at 10Hz as that appears to be the default MIRA value, but we can change it. It is set in the file `scitos_mira/resources/SCITOS-G5.xml`, line 10.

If you test and find that 30Hz, or even 100Hz has no negative effect on anything, we could change the repo. Alternatively, you could run scitos_mira with a different config file of your own by setting the  parameter config_file (scitos_mira node space parameter) to your version. In the long run though, it is best to have us all using the same frequency....

**Comment by cdondrup** [comment]
Thanks for the tip. I will test it over next few days see if a higher frequency causes any problems.

**Comment by cdondrup** [comment]
I just played with the parameters a bit and noticed some weird behaviour:
- If you set it to a value lower than 20ms, it won't publish anything (I only tried increments of 5 so anything between 15 and 20 might also work)
- The publish rate always starts to publish at the hz rate it last did before killing the scitos_node and then goes up very slowly. Took about 5 minutes to get up from 10hz to 30hz. It now seems to have stabilised at ~47hz

We will let it run with these parameters and see if it has any negative effects.

Btw: @cburbridge what does line 11: `<Force>80</Force>` mean? Does this "force" have to do anything with the odometry?
